### PR TITLE
rough pep8 cleanup, remove lambdas, ...

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

File: plone/app/content/namechooser.py [hunk 1]
```diff
@@ -96,12 +95,12 @@ def _getCheckId(self, obj):
...
-            check_id = lambda id, required: do_Plone_check(id, required)
+            check_id = do_Plone_check
         else:
             def do_OFS_check(parent, id):
                 try:
                     parent._checkId(id)
                 except BadRequest:
                     return True
-            check_id = lambda id, required: do_OFS_check(parent, id)
+            check_id = do_OFS_check
```
Comment (davisagli) [inline plone/app/content/namechooser.py]: Careful; do_OFS_check does not take the same parameters as the lambda

**Comment by jensens** [comment]
jenkins at http://jenkins.plone.org/view/All/job/pull-request/35/

File: plone/app/content/namechooser.py (hunk 1 above)
Comment (jensens) [inline plone/app/content/namechooser.py]: indeed. i'll fix

**Comment by jensens** [comment]
runs again http://jenkins.plone.org/view/All/job/pull-request/36/

**Comment by jensens** [comment]
@tisto the above job fails because it cant find plone.app.contentrules in sources. Is it possible there are some cross-test effects?

**Comment by jensens** [comment]
started again with fix by @tisto http://jenkins.plone.org/view/All/job/pull-request/39/console

**Comment by jensens** [comment]
PR test is green - anybody for a quick review and merge?

**Pull request merged**